Makefile

```diff
@@ -0,0 +1,8 @@
+down:
+	docker-compose down
+
+build:
+	docker-compose build
+
+serve: down build
```

Comment:
If this were moved to the top it becomes the default when you run `make` without any args.

Reply:
Should probs comment this over on https://github.com/robertjlooby/elm-koans/pull/26 but yeah - I agree

I’ve seen some people comment on other repos that `make` with no args doesn’t compile the app. I find this a sensible although somewhat difficult interface to maintain liskov compatibility with (when considering Makefile in a Ruby project)

Reply:
(As in, make with no arguments not compiling is weird)